test/test_properties.py

```diff
@@ -10,6 +10,8 @@ def test_event_has_summary(calendars):
 def test_recurrent_events_change_start_and_end(calendars, attribute):
     events = calendars.three_events_one_edited.all()
     print(events)
+    if events[0][attribute].__hash__ is None:
```

Comment:
no. The test becomes worthless.

Reply:
> I sent a few comments. I might chip in some code later but am quite busy, too.

It's good to coordinate that we don't work at the same things at the same time :-) I will apparently be busy for the rest of the day.